docs/content/getting-started/setup-openfga.mdx

```diff
@@ -165,7 +165,7 @@ If you are going to use this setup in production, you should enable TLS in your
 
 The Playground facilitates rapid development by allowing you to visualize and model your application's authorization model(s) and manage relationship tuples with a locally running OpenFGA instance.
 
-The Playground is enabled on port 3000 by default. To disable the Playground, please take a look at [Configuring the Server](#configuring-the-server) for more information.
+The Playground is enabled on port 3000 by default (if using the docker or docker-compose examples above, the url will be http://localhost:3000/playground). To disable the Playground, please take a look at [Configuring the Server](#configuring-the-server) for more information.
```

Comment:
```suggestion
The Playground is enabled on port 3000 by default and accessible at http://localhost:3000/playground (if using the docker make sure port 3000 is exposed to your local network). To disable the Playground, please take a look at [Configuring the Server](#configuring-the-server) for more information.
```

Reply:
Updated the phrasing to make it clearer that this also applies to non-docker installation methods too.